roles/opendistro/opendistro-elasticsearch/tasks/main.yml

```diff
@@ -64,7 +64,7 @@
       return_content: yes
       timeout: 4
     register: _result
-    until: (  _result.json is defined) and (_result.json.status == "green")
+    until: (  _result.json is defined) and (_result.json.status == "green" or _result.json.status == "yellow" )
```

Comment:
Should we also check for single node condition here?

```yaml
until: (  _result.json is defined) and (_result.json.status == "green" or (_result.json.status == "yellow" and single_node) )
```

Reply:
I agree, this way we ensure that the check is successful when the health of the cluster is yellow only for single-node configurations.

Fixed here d23feddedec43a1ae37b956ec66241c62249fb7d